Fix typo and improve writing in Multiple resolutions (#3644)
godotengine/godot-docs: tutorials/viewports/multiple_resolutions.rst

Stretch settings
----------------

> Stretch settings are located in the project settings. They're just a bunch

Reviewer: We can simplify this bit by removing filler words:

```suggestion
Stretch settings are located in the project settings and provide several options:
```

> of configuration variables that provide several options:

Reviewer: ```suggestion
```
